Marlin/ubl_motion.cpp

```diff
@@ -639,13 +643,14 @@
         const float z_xmy0 = (z_x1y0 - z_x0y0) * (1.0 / (MESH_X_DIST)),   // z slope per x along y0 (lower left to lower right)
                     z_xmy1 = (z_x1y1 - z_x0y1) * (1.0 / (MESH_X_DIST));   // z slope per x along y1 (upper left to upper right)
 
-              float z_cxy0 = z_x0y0 + z_xmy0 * cx;          // z height along y0 at cx
+              float z_cxy0 = z_x0y0 + (z_xmy0 * cx);          // z height along y0 at cx
```

Comment:
You had made other edits to my code that ADDED parentheses to expression like this, so I was trying to be consistent.  I preferred the original way.   Which way do you want it?